From b41b373859d70c056d50edea7e9120a8bc5943f4 Mon Sep 17 00:00:00 2001
From: Carlos Rafael Giani <dv@pseudoterminal.org>
Date: Sun, 25 Feb 2018 14:16:44 +0100
Subject: [PATCH] gl: Add DRM cflags

This prevents cross compilation errors like:

usr/include/xf86drm.h:40:10: fatal error: drm.h: No such file or directory

These are caused because the GBM display header includes xf86drm.h
---
 gst-libs/gst/gl/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gst-libs/gst/gl/Makefile.am b/gst-libs/gst/gl/Makefile.am
index 3ec615240..7779bc0e4 100644
--- a/gst-libs/gst/gl/Makefile.am
+++ b/gst-libs/gst/gl/Makefile.am
@@ -154,6 +154,7 @@ libgstgl_@GST_API_VERSION@_la_CFLAGS = \
 	$(GST_PLUGINS_BASE_CFLAGS) \
 	$(GST_BASE_CFLAGS) \
 	$(GST_CFLAGS) \
+	$(DRM_CFLAGS) \
 	$(GL_CFLAGS)
 
 libgstgl_@GST_API_VERSION@_la_LDFLAGS = \
-- 
2.14.1

